src/test/java/roomescape/MemberTest.java

```diff
@@ -0,0 +1,57 @@
+package roomescape;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.DirtiesContext;
+import roomescape.member.ui.dto.MemberRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT, properties = {"server.port=8888"})
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class MemberTest {
+    @BeforeEach
+    private void setPort() {
+        RestAssured.port = 8888;
+    }
```

Comment:
```suggestion
    @BeforeEach
    public void setPort() {
        RestAssured.port = 8888;
    }
```

해당 메서드는 private 이 아닌, public 으로 바꿔주면 좋을 것 같아요.
다른 테스트 클래스도 한 번 확인해주세요!

Reply:
public으로 선언해야하는 이유가 있나요??

Reply:
JUnit 은 테스트 라이프사이클을 관리하기 위해 리플렉션을 사용하는데요,
public 으로 두면 JUnit 이 테스트 실행 위치 (패키지, 클래스) 에 상관없이 이를 읽어들일 수 있어요.
intelliJ 자동완성 기능으로 해당 메서드를 만들면 알아서 public 으로 만들어주기도 해서 편해요!